src/components/organisms/top/top.html

```diff
@@ -0,0 +1,17 @@
+<div class="top">
+  <div class="top-wrap">
+    <img src="../../../img/top.jpg" alt="" />
+    <div class="front-message">
+      <h2 class="title">そうだ。<br />カフェに行こう。</h2>
+    </div>
+  </div>
+  <div class="advertisement">
+    <h2 class="title">
+      カフェで読書＆コワーキング。快適のフロアスペース空間。
+    </h2>
+    <p class="explanation">
+      カフェエリアでコーヒーを注文し、ブックエリアでお好みの本を選び、コワーキングスペースでコーヒーと共に読書を楽しむ。<br />
+      学校・仕事帰りに、平日の日中に、休日にゆっくり...さまざまなライフスタイルに合わせてご利用いただける多彩なプランをご用意しております。
+    </p>
+  </div>
```

Comment:
ここは過去に作った.messageを使うと良いところです
```suggestion
  <div class="advertisement">
  <div class="message">
    <h2>
      カフェで読書＆コワーキング。快適のフロアスペース空間。
    </h2>
    <p>
      カフェエリアでコーヒーを注文し、ブックエリアでお好みの本を選び、コワーキングスペースでコーヒーと共に読書を楽しむ。<br />
      学校・仕事帰りに、平日の日中に、休日にゆっくり...さまざまなライフスタイルに合わせてご利用いただける多彩なプランをご用意しております。
    </p>
</div>
  </div>
```